docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,92 @@
+
+# 
+
+1. make sure `# kubectl api-versions` lists `apiregistration.k8s.io/v1beta1`
+
+# this won't show up if you are talking to the insecure port. the insecure port is not behind the aggregator.
+
+# Get the appropriate tls ca/cert/keys from kube
+# TODO how do I know which ones to get? Are they ones given to specific flags on the apiserver, etc etc?
```

Comment:
You don't *need* to use the main CA, but if you want to, it's the cert passed with the `--root-ca-file` flag to the controller-manager.